mlflow/langchain/databricks_dependencies.py

```diff
@@ -273,6 +273,39 @@ def _traverse_runnable(
         pass
 
 
+def _get_deps_from_closures(lc_model):
+    """
+    In some cases, the dependency extraction of Runnable Lambda fails because the call
```

Comment:
While it's 100% fine to release this patch in MLflow now to unblock, I feel this fix should reside in LangChain side, because this is a bug in their code that has no dependency to MLflow. Could you open an issue in their repo and add TODO comment linking to it?